scripts/test-integration.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+projectRoot="$_dir/.."
+exDevConfigFile="$projectRoot/config/config.dev.json.example"
+devConfigFile="$projectRoot/config/config.dev.json"
+
+if [[ ! -f "$devConfigFile" ]]; then
+  cp "$exDevConfigFile" "$devConfigFile"
+fi
+# TODO: This server dest+port should be parameterized somehow
+./node_modules/.bin/start-server-and-test start http://localhost:9000 './node_modules/.bin/cypress run'
```

Comment:
You still need to use the `$projectRoot` here: https://github.com/eriknelson/mig-ui/blob/andyvesel-add-tests/scripts/test-integration.sh#L12